plugins/scaffolder/src/alpha/components/TemplateEditorPage/TemplateEditorToolbar.tsx

```diff
@@ -120,32 +131,37 @@ export function TemplateEditorToolbar(props: {
           aria-labelledby="publish-dialog-title"
           aria-describedby="publish-dialog-description"
         >
-          <DialogTitle id="publish-dialog-title">Publish changes</DialogTitle>
+          <DialogTitle id="publish-dialog-title">
+            {t('templateEditorToolbar.addToCatalogDialogTitle')}
+          </DialogTitle>
           <DialogContent dividers>
             <DialogContentText id="publish-dialog-slide-description">
-              Follow the instructions below to create or update a template:
+              {t(
+                'templateEditorToolbar.addToCatalogDialogContent.introduction',
+              )}
               <ol>
-                <li>Save the template files in a local directory</li>
                 <li>
-                  Create a pull request to a new or existing git repository
+                  {t('templateEditorToolbar.addToCatalogDialogContent.step1')}
                 </li>
                 <li>
-                  If the template already exists, the changes will be reflected
-                  in the software catalog once the pull request gets merged
+                  {t('templateEditorToolbar.addToCatalogDialogContent.step2')}
                 </li>
                 <li>
-                  But if you are creating a new template, follow this{' '}
+                  {t('templateEditorToolbar.addToCatalogDialogContent.step3')}
+                </li>
+                <li>
+                  {t('templateEditorToolbar.addToCatalogDialogContent.step4')}
+                  <br />
                   <Link to="https://backstage.io/docs/features/software-templates/adding-templates/">
-                    documentation
-                  </Link>{' '}
-                  to register the new template repository in software catalog
+                    https://backstage.io/docs/features/software-templates/adding-templates/
+                  </Link>
```

Comment:
We'll need to make this link customizable for this to be useful imo. I don't think there's any way to do that right now right?

Reply:
I added the link to the translation keys, is that a safe solution for now?